### PR TITLE
CHEMH-186: updates to text link styles

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Updates to link styles.
- remove the background color
- changed the hover to the dark cardinal red

# Review By (Date)
- soon

# Criticality
- normal

# Urgency
- normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to page and look at links. 
4. Verify links are blue and then red on hover and focus. Verify the icons follow suit. 

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- CHEMH-186](https://stanfordits.atlassian.net/browse/CHEMH-186)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by jdwjdwjdw — CHANGES_REQUESTED** [review]
Can you update the link icon to change colors on active everywhere as well?
Screen Shot 2023-05-04 at 4 55 52 PM

Also, given these requested updates and our zoom conversation this morning, I'm not sure what the desired colors for the CTA Action link are. On this PR, it's blue then red on hover, similar to regular links. [Figma](https://www.figma.com/proto/wq9Y5igro1BLYN0kbeojC7/ChEM-H-Design-02?node-id=874-5538) shows black then blue on hover, but again, with the requested updates, maybe this is the preferred way. So let me know your thoughts on that.

**Comment by jenbreese** [comment]
@jdwjdwjdw I do not know what  she'd like for the CTA action links. I did fix the `:active`. This change was only for the footer and the body copy changes. I made the cta link match the design. That seemed the best action.

File: src/scss/components/local-footer/_local-footer.scss [hunk 1]
```diff
@@ -239,15 +241,19 @@
...
   a,
   a.su-link--external,
-  a.su-link {
+  a.su-link,
+  .su-local-footer__action-links {
     color: $chemh-color-digital-blue;
     text-decoration: underline;
 
     &:hover,
     &:focus {
```
Comment (jdwjdwjdw) [inline src/scss/components/local-footer/_local-footer.scss]: Can you adjust this for active as well? Other changes look good!
image

**Review by jdwjdwjdw — APPROVED** [review]
Looks great, thanks @jenbreese!

**Pull request merged**